### PR TITLE
Docs Update - Eslint Config README w/example override

### PR DESCRIPTION
**Pull request by webstackdev**
Added info and example on overriding ESLint config, in response to Forum question [#2275](https://community.redwoodjs.com/t/redwood-eslint-config-modifications/2275)

### CODE REVIEW

File: packages/eslint-config/README.md [hunk 1]
```diff
@@ -1,28 +1,73 @@
...
-2) How this package fits into the Redwood Framework, if it depends on other Redwood packages, etc.
-3) The structure of the package and/or an explanation of its contents.
+This package has no dependencies on other Redwood Framework packages. To contribute, you should be familiar with the ESLint package. Keep in mind that any rules added should not conflict with code formatting tools (e.g. [Prettier](https://prettier.io/docs/en/integrating-with-linters.html)).
+
+## Overriding Default Configuration
+
+In a Redwood App, you can override default config in your root `package.json` file by adding the rules after the include for this package:
+
+```javascript
+// redwood/package.json
```
Comment (jtoar) [inline packages/eslint-config/README.md:41]: - just a little clearer
```suggestion
// redwood-app/package.json
```

File: packages/eslint-config/README.md [hunk 2]
```diff
@@ -1,28 +1,73 @@
...
+
+- Lint for case-sensitive imports (issue [#2806](https://github.com/redwoodjs/redwood/issues/2806))
 
 ## Contributing
-Explains how to contribute by addressing the following three points:
 
-1) Core technologies a contributor should be a familiar with.
-2) How this package fits into the Redwood Framework, if it depends on other Redwood packages, etc.
-3) The structure of the package and/or an explanation of its contents.
+This package has no dependencies on other Redwood Framework packages. To contribute, you should be familiar with the ESLint package. Keep in mind that any rules added should not conflict with code formatting tools (e.g. [Prettier](https://prettier.io/docs/en/integrating-with-linters.html)).
```
Comment (jtoar) [inline packages/eslint-config/README.md:34]: - sounded a little awkward
```suggestion
This package doesn't depend on other Redwood Framework packages. To contribute, you should be familiar with the ESLint package. Keep in mind that any rules added should not conflict with code formatting tools (e.g. [Prettier](https://prettier.io/docs/en/integrating-with-linters.html)).
```

File: packages/eslint-config/README.md [hunk 3]
```diff
@@ -1,28 +1,73 @@
...
+  "extends": "@redwoodjs/eslint-config",
+  "root": true,
+  "jsx-a11y/no-onchange": "off",
+},
+```
+
+If you need script in your configuration, you can remove the `eslintConfig` block from your root `package.json` file and add an `.eslintrc.js` file:
+
+```javascript
+// redwood/.eslintrc.js
```
Comment (jtoar) [inline packages/eslint-config/README.md:52]: - just a little clearer
```suggestion
// redwood-app/.eslintrc.js
```

File: packages/eslint-config/README.md [hunk 4]
```diff
@@ -1,28 +1,73 @@
...
+module.exports = {
+  extends: ['@redwoodjs/eslint-config'],
+  'root': true,
+  rules: {
+    'jsx-a11y/no-onchange': 'off',
+  },
+}
+```
+
+By default, ESLint will recurse through all project directories looking for configuration files and directives, and overriding those specified in multiple places according to a prioritization formula. The `root` directive tells ESLint to stop searching for configuration lower in the tree at the file this directive is encountered.
```
Comment (jtoar) [inline packages/eslint-config/README.md:62]: - tense maybe?
```suggestion
By default, ESLint will recurse through all project directories looking for configuration files and directives, and override those specified in multiple places according to a prioritization formula. The `root` directive tells ESLint to stop searching for configuration lower in the tree at the file this directive is encountered.
```

**Review by jtoar — CHANGES_REQUESTED** [review]
@webstackdev this is fantastic, thank you so much! I suggested a few changes for clarity, let me know what you think

**Comment by webstackdev** [comment]
@jtoar thanks, your suggests are added to the PR

**Review by jtoar — APPROVED** [review]

**Pull request merged**